docs/bucket/notifications/README.md

```diff
@@ -231,54 +272,54 @@ client.connect("localhost",1883,60)
 client.loop_forever()
 ```
 
-Execute this example python program to watch for MQTT events on the console.
+Use the following command to execute this program and watch for MQTT events on the console:
 
 ```py
 python mqtt.py
 ```
 
-Open another terminal and upload a JPEG image into ``images`` bucket.
+Open another console and use the following command to upload a JPEG image into ``images`` bucket:
 
-```
+```sh
 mc cp myphoto.jpg myminio/images
 ```
 
-You should receive the following event notification via MQTT once the upload completes.
+Once the upload completes, return to the first console. A response similar to this one should be displayed showing an event notification from MQTT:
 
-```py
-python mqtt.py
-{“Records”:[{“eventVersion”:”2.0",”eventSource”:”aws:s3",”awsRegion”:”",”eventTime”:”2016–09–08T22:34:38.226Z”,”eventName”:”s3:ObjectCreated:Put”,”userIdentity”:{“principalId”:”minio”},”requestParameters”:{“sourceIPAddress”:”10.1.10.150:44576"},”responseElements”:{},”s3":{“s3SchemaVersion”:”1.0",”configurationId”:”Config”,”bucket”:{“name”:”images”,”ownerIdentity”:{“principalId”:”minio”},”arn”:”arn:aws:s3:::images”},”object”:{“key”:”myphoto.jpg”,”size”:200436,”sequencer”:”147279EAF9F40933"}}}],”level”:”info”,”msg”:””,”time”:”2016–09–08T15:34:38–07:00"}
+```json
+{“Records”:[{“eventVersion”:”2.0",”eventSource”:”aws:s3",”awsRegion”:”",”eventTime”:”2016–09–08T22:34:38.226Z”,”eventName”:”s3:ObjectCreated:Put”,
+ ”userIdentity”:{“principalId”:”minio”}, ”requestParameters”:{“sourceIPAddress”:”10.1.10.150:44576"},”responseElements”:{},”s3":
+ {“s3SchemaVersion”:”1.0",”configurationId”:”Config”,”bucket”:{“name”:”images”,”ownerIdentity”:{“principalId”:”minio”},”arn”:”arn:aws:s3:::images”},
+”object”:{“key”:”myphoto.jpg”,”size”:200436,”sequencer”:”147279EAF9F40933"}}}],”level”:”info”,”msg”:””,”time”:”2016–09–08T15:34:38–07:00"}
 ```
 
-<a name="Elasticsearch"></a>
-## Publish Minio events via Elasticsearch
-
-Install [Elasticsearch](https://www.elastic.co/downloads/elasticsearch) server.
+## 4.<a name="publish-minio-events-via-elasticsearch"></a> Publish Minio Events via Elasticsearch
 
-This notification target supports two formats: _namespace_ and _access_.
+### 4.1 Install Elasticsearch
 
-When the _namespace_ format is used, Minio synchronizes objects in the bucket with documents in the index. For each event in the Minio, the server creates a document with the bucket and object name from the event as the document ID. Other details of the event are stored in the body of the document. Thus if an existing object is over-written in Minio, the corresponding document in the Elasticsearch index is updated. If an object is deleted, the corresponding document is deleted from the index.
+Minio requires Elasticsearch 5.x which is the latest major release series.
```

Comment:
AFAIK the latest version of ES is 6.5, we should remove any reference to "latest" as that will not age well.  @harshavardhana did we verify that these instructions work with the latest version?

Reply:
No, we cannot guarantee, but anything 5.x is okay.